twoliter/src/cmd/build.rs

```diff
@@ -13,18 +13,84 @@ use tempfile::TempDir;
 #[derive(Debug, Parser)]
 pub(crate) enum BuildCommand {
     Clean(BuildClean),
+    Kit(BuildKit),
     Variant(BuildVariant),
 }
 
 impl BuildCommand {
     pub(crate) async fn run(self) -> Result<()> {
         match self {
             BuildCommand::Clean(command) => command.run().await,
+            BuildCommand::Kit(command) => command.run().await,
             BuildCommand::Variant(command) => command.run().await,
         }
     }
 }
 
+/// Build a Bottlerocket variant image.
+#[derive(Debug, Parser)]
+pub(crate) struct BuildKit {
+    /// Path to Twoliter.toml. Will search for Twoliter.toml when absent.
+    #[clap(long = "project-path")]
+    project_path: Option<PathBuf>,
+
+    /// The architecture to build for.
+    #[clap(long = "arch", default_value = "x86_64")]
+    arch: String,
```

Comment:
Can we default to the arch that `twoliter` was built for? It'd be sort of surprising to get x86_64 builds by default on an aarch64 host.

Reply:
Will do, #253